### ISSUE TITLE
“TrackerD v1.3.layout.pdf” and “TrackerD v1.3.silkscreen.pdf”  missing

### ISSUE THREAD

**Issue by umsenn** [dragino/TrackerD#34]
The hardware documents in [https://github.com/dragino/TrackerD/tree/main/hardware](https://github.com/dragino/TrackerD/tree/main/hardware) do not represent the actual v1.3 hardware release.

The v1.3 schematic diagram is also missing there but can be found using the corresponding shared folder link on the dragino website https://www.dragino.com/products/tracker/item/234-trackerd.html .

**Comment by namejinnameyuan** [comment]
Yes, the new design has not been updated to the above, only the interrupt IO has been changed. The new version of the hardware has an interrupt IO of pin 25.
